src/main/java/org/wso2/carbon/inbound/iso8583/listening/ConnectionRequestHandler.java

```diff
@@ -120,4 +126,4 @@ private void handleException(String msg, Exception e) {
         log.error(msg, e);
```

Comment:
there’s a chance that the exception will be logged again further up the call stack, resulting in two log events for the same error. It seems like this was not your commit.